zlib/example_test.go

```
@@ -18,8 +17,19 @@ func ExampleNewWriter() {
-	// Output: [120 156 202 72 205 201 201 215 81 40 207 47 202 73 225 2 4 0 0 255 255 33 231 4 147]
+
+	// The stdlib flate encoder's exact byte output for a given input is not
+	// part of the Go compatibility promise (only decodability is), and has
+	// changed between Go versions (e.g. its tiny-input store-vs-compress
+	// heuristic). So round-trip through a Reader instead of asserting the
+	// compressed bytes verbatim.
+	r, err := zlib.NewReader(&b)
+	if err != nil {
+		panic(err)
+	}
+	io.Copy(os.Stdout, r)
```

golangci-lint (2.12.2)

[error] Error return value of `io.Copy` is not checked

(errcheck)